opentelemetry/proto/collector/trace/v1/trace_service_http.yaml

```diff
@@ -5,8 +5,5 @@ config_version: 3
 http:
  rules:
  - selector: opentelemetry.proto.collector.trace.v1.TraceService.Export
-   post: /v1/trace
```

Comment:
Not sure why `/v1/trace1` was mapped to MetricsService?
The code was moved from `opentelemetry-java` (https://github.com/open-telemetry/opentelemetry-java/issues/235), so it's hard to track the history of this change.